docs/conf.py

```diff
@@ -0,0 +1,35 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'ReX'
+copyright = '2024, David Kelly'
+author = 'David Kelly'
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    'autoapi.extension',
+    'sphinx.ext.autodoc.typehints',
+    'myst_parser',
+    'sphinx.ext.napoleon'
+]
+
+templates_path = ['_templates']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = 'alabaster'
+html_static_path = ['_static']
+
+# -- AutoAPI -----------------------------------------------------------------
+# https://sphinx-autoapi.readthedocs.io/en/latest/
+autoapi_dirs = ['../rex_ai']
+autodoc_typehints = 'description'
```

Comment:
```suggestion
autodoc_typehints = 'description'

```
Tiny fix for lack of newline at end of file.